addons/web/static/src/scss/primary_variables.scss

```diff
@@ -287,3 +287,5 @@ $o-btns-bs-outline-override: map-merge((
         active-color: $o-component-active-color,
     ),
 ), $o-btns-bs-outline-override);
+
+$form-range-thumb-focus-box-shadow: 0 0 0 1px $o-brand-primary;
```

Comment:
Hello @brboi , thanks for this fix 👍 

I'm ok with the change, except that the variable should be defined in `addons/web/static/src/scss/bootstrap_overridden.scss` that's where Bootstrap's default variables are normally customised (there should be something related to range inputs already). 

 Don't forget to add the `!default` flag there in order to ease bundle's customizations for POS,  barcode scanner and others...
 
 You my find this interesting: https://www.odoo.com/documentation/master/developer/reference/user_interface/scss_inheritance.html?highlight=inheritance#scss-s-default-directive

Thanks again 👍